### DISCUSSION TITLE
肥肠感谢，bml买上了

### DISCUSSION THREAD

**Discussion by Terminal-Void** [mikumifa/biliTickerBuy#293] [聊天专区]
感谢

**Comment by Kevin98521** [comment]
怎么设置的求教学

**Reply by sunnyyangyangyang** [reply]
https://github.com/mikumifa/biliTickerBuy/discussions/274#discussion-6877406

**Reply by sunnyyangyangyang** [reply]
[fyydvay](https://github.com/fyydvay)

B区和C区最好抢
本地过验证
间隔300
开票前10s启动
我零点几秒就抢到了14号C区的

**Comment by sunnyyangyangyang** [comment]
感谢
